tutorials/2d/2d_lights_and_shadows.rst

```diff
@@ -0,0 +1,226 @@
+blending to add the color of their texture to the scene. 
+
+.. image:: img/light_shadow_light.png
+
+:ref:`Lights <class_Light2D>` have four ``modes``: ``add``, ``sub``, ``mix``, and ``mask``.
+
+``Add`` adds the color of the light texture to the scene. It brightens the area under the light.
+
+``Sub`` subtracts the color of the light from the scene. It darkens the area under the light.
+
+``Mix`` mixes the color of the light with the underlying scene. The resulting brightness is
+halfway between the color of the light and the color underneath.
+
+``Mask`` is used to mask out areas that are covered by the light. Masked out areas are hidden or revealed based on 
+the color of the light.
+
+For the demo the lights have two components, the :ref:`Light <class_Light2D>` itself (which 
+is the effect of the light), and a :ref:`Sprite <class_Sprite>` blob which is just an image that shows the 
+location of the light source. A child :ref:`Sprite <class_Sprite>` is not necessary to make a 
+:ref:`Light <class_Light2D>` work but is used to show the location of the light.
+
+.. image:: img/light_shadow_light_blob.png
+
+Shadows
+-------
+
+Shadows are made by intersecting a :ref:`Light <class_Light2D>` with a :ref:`LightOccluder2D <class_LightOccluder2D>`.
+
+By default shadows are turned off. To turn them on click on the :ref:`Light <class_Light2D>` 
+and under the Shadows section check ``Enabled``.
+
+In the demo we are using a :ref:`Sprite <class_Sprite>` with a Texture on it to make the "Shadow Casters", 
+but in reality all you need is a couple :ref:`LightOccluder2Ds <class_LightOccluder2D>`. By itself 
+the :ref:`LightOccluder2D <class_LightOccluder2D>` looks like a dark spot and in this demo the :ref:`Sprite <class_Sprite>` is 
+just a black square.
+
+Step by step
+------------
+
+Now that we have covered the basics of the nodes that will be used, we will now walk step by step through 
+the process of making a scene like the one found in the demo.
+
+First add a :ref:`Sprite <class_Sprite>` and set its texture to the `background image <https://raw.githubusercontent.com/godotengine/godot-demo-projects/master/2d/lights_and_shadows/bg.png>`_. For your game this can be whatever 
+your background will be. For this style of shadows it is most likely to be a floor texture. 
+
+.. image:: img/light_shadow_background.png
+
+Next create three :ref:`Light2D's <class_Light2D>` and set their textures to the `light image <https://raw.githubusercontent.com/godotengine/godot-demo-projects/master/2d/lights_and_shadows/light.png>`_. You can alter their 
+color in the top section. By default shadows are turned off and the ``mode`` is set to ``add``. This 
+means that each light adds its own color to whatever is underneath.
+
+.. image:: img/light_shadow_all_lights_no_blob.png
+
+Next child a :ref:`Sprite <class_Sprite>` to each of the :ref:`Light <class_Light2D>` nodes, and set 
```

Comment:
I'm not fond of using "child" as a verb. I'd prefer "add a Sprite" or "add a child Sprite"